packages/ui/src/forum/components/category/PostInfo.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+import styled from 'styled-components'
+
+import { TextMedium, TextSmall } from '@/common/components/typography'
+import { relativeTime } from '@/common/model/relativeTime'
+import { spacing } from '@/common/utils/styles'
+import { ForumPost } from '@/forum/types'
+import { MemberInfo } from '@/memberships/components'
+
+interface PostInfoProps {
+  post: ForumPost
+}
+export const PostInfo = ({ post }: PostInfoProps) => (
+  <PostInfoStyles>
+    <TextMedium as="h6" bold>
+      {post.text.slice(0, 100)}
+    </TextMedium>
+    <TextSmall as="div" lighter>
```

Comment:
Use CategorListItemText. Or you can add TextExtraSmall into /typography/Text.tsx